prow/genjobs/main.go

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2019 Istio Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"istio.io/test-infra/prow/genjobs/cmd/genjobs"
+)
+
+func main() {
+	genjobs.Main()
```

Comment:
What's the motivation here? None of our other utilities have this pattern.

Is there some reason people can't call k8s.io/prow/cmd/genjobs like all the other tools (tackle, etc)

Reply:
I stole this pattern from [`kind`](https://github.com/kubernetes-sigs/kind/blob/master/main.go). I will change this if you are adamantly opposed.